DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -328,6 +336,22 @@ static ExpressionSyntax CreateStringExpressionSyntax(string str)
                     SyntaxFactory.IdentifierName(typeof(T).FullName),
                     SyntaxFactory.IdentifierName(value.ToString()));
             }
+
+            static bool IncludeArgInDllImportAttribute(string argName)
```

Comment:
Can we update this to be `PassThroughToDllImportAttribute`? The term "PassThrough" is clearer to me.

Reply:
This also seems like something we may want to document in our design? Seems like we are maintaining compat but how we maintain that is a design detail I would like to see reasoned about.

Reply:
Yeah, I was thinking we should have a section specifically calling out each attribute (e.g. dropping things like this, always adding `EntryPoint`).

Planning on putting it in the `P/Invoke` section in `Pipeline.md` - unless folks think it belongs in the compat doc / elsewhere.